packages/pluggableWidgets/maps-native/src/Maps.xml

```diff
@@ -1,69 +1,168 @@
-                                <description>When latitude and longitude are provided, the address is ignored.</description>
-                                <returnType type="String"/>
-                            </property>
-                            <property key="latitude" type="expression" required="false">
-                                <caption>Latitude</caption>
-                                <description/>
-                                <returnType type="Decimal"/>
-                            </property>
-                            <property key="longitude" type="expression" required="false">
-                                <caption>Longitude</caption>
-                                <description/>
-                                <returnType type="Decimal"/>
-                            </property>
-                        </propertyGroup>
-                        <propertyGroup caption="Callout box">
-                            <property key="title" type="textTemplate" required="false">
-                                <caption>Title</caption>
-                                <description/>
-                            </property>
-                            <property key="description" type="textTemplate" required="false">
-                                <caption>Description</caption>
-                                <description/>
-                            </property>
+        <propertyGroup caption="General">
+            <propertyGroup caption="Markers">
+                <property key="markers" type="object" isList="true" required="false">
+                    <caption>Markers</caption>
+                    <description>A list of static locations on the map.</description>
+                    <properties>
+                        <propertyGroup caption="Locations">
+                            <propertyGroup caption="Location">
+                                <property key="locationType" type="enumeration" defaultValue="address">
+                                    <caption>Location</caption>
+                                    <description/>
+                                    <enumerationValues>
+                                        <enumerationValue key="address">Based on address</enumerationValue>
+                                        <enumerationValue key="latlng">Based on latitude and longitude
+                                        </enumerationValue>
+                                    </enumerationValues>
+                                </property>
+                                <property key="address" type="textTemplate" required="false">
+                                    <caption>Address</caption>
+                                    <description>Address containing (a subset of) street, number, zipcode, city and country.</description>
+                                </property>
+                                <property key="latitude" type="textTemplate" required="false">
```

Comment:
Why latitude is now text template? This widget is not intended for Studio (which does not support expressions)

Reply:
I thought it would be a good idea to align with web widget. Should all textTemplates on native be converted to expression or ?

Reply:
I think we should not use string for latitude and longitude, specially for native which is designed for SP only